util/execdetails.go

```diff
@@ -255,24 +362,50 @@ func (sd *ScanDetail) String() string {
 	}
 	buf := bytes.NewBuffer(make([]byte, 0, 16))
 	buf.WriteString("scan_detail: {")
-	buf.WriteString("total_process_keys: ")
-	buf.WriteString(strconv.FormatInt(sd.ProcessedKeys, 10))
-	buf.WriteString(", total_process_keys_size: ")
-	buf.WriteString(strconv.FormatInt(sd.ProcessedKeysSize, 10))
-	buf.WriteString(", total_keys: ")
-	buf.WriteString(strconv.FormatInt(sd.TotalKeys, 10))
+	if sd.ProcessedKeys > 0 {
+		buf.WriteString("total_process_keys: ")
+		buf.WriteString(strconv.FormatInt(sd.ProcessedKeys, 10))
+	}
+	if sd.ProcessedKeysSize > 0 {
+		buf.WriteString(", total_process_keys_size: ")
+		buf.WriteString(strconv.FormatInt(sd.ProcessedKeysSize, 10))
+	}
+	if sd.TotalKeys > 0 {
+		buf.WriteString(", total_keys: ")
+		buf.WriteString(strconv.FormatInt(sd.TotalKeys, 10))
+	}
+	if sd.GetSnapshotDuration > 0 {
+		buf.WriteString("get_snapshot_time: ")
```

Comment:
```suggestion
		buf.WriteString(", get_snapshot_time: ")
```

I'm afraid the not-well formatted output may break some analysis tools.